voluptuous/validators.py

```diff
@@ -794,3 +794,57 @@ def __call__(self, v):
 
     def __repr__(self):
         return 'Unordered([{}])'.format(", ".join(repr(v) for v in self.validators))
+
+
+
+class Number(object):
+    """
+    Verify the maximum number of digits that are present in the number(Precision),
+    and the decimal places(Scale)
+
+    :raises Invalid: If the value does not match the provided Precision and Scale.
+
+    >>> schema = Schema(Number(precision=6, scale=2))
+    >>> schema('1234.01')
+    Decimal('1234.01')
```

Comment:
@nareshnootoo  Let's add a flag which tells what to yield, where default being the string since we are giving input as string only.